docs/api/datasets/occupancy.rst

```diff
@@ -0,0 +1,23 @@
+.. -*- mode: rst -*-
```

Comment:
This is a detail for a classification dataset - but note that we have much less information in this detail than we did in Bikeshare just because of the differences in their READMEs.